hrpsys_ros_bridge/euslisp/rtm-ros-robot-interface.l

```diff
@@ -329,7 +329,7 @@
         (debug nil))
   (when (boundp param-class)
     (let* ((param-slots-list ;; get slots list for param-class
-            (remove-if #'(lambda (x) (string= "plist" x))
+            (remove-if #'(lambda (x) (or (string= "plist" x) (string= "connection-header")))
```

Comment:
`(string= "connection-header")` -> `(string= "connection-header" x)` ???
Sorry for trouble

Reply:
Yes, I was wrong. (string= "connection-header" x) is correct.